appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -1942,11 +1970,18 @@ private void configureHazelcast() {
             System.setProperty("hazelcast.wait.seconds.before.join", Integer.toString(initialJoinWait));
         }
 
-        if (noCluster) {
+        if (noCluster)
+        {
+            /*preBootCommands.add(new BootCommand("set",
+              "configs.config.server-config.hazelcast-config-specific-configuration.enabled=false"));
+            preBootCommands.add(new BootCommand("set",
+              "configs.config.server-config.ejb-container.ejb-timer-service.ejb-timer-service=Dummy"));*/
+            System.setProperty("payaramicro.noCluster", String.valueOf(true));
```

Comment:
I wouldn't do this as a property.
This strikes me as something that should be done in the same way as noHazelcast - persisted to the domain.xml / server config

Reply:
I can't do it as a suggestion, but you'll need to add the following entries to the `HazelcastConfigSpecificConfiguration` class, and then the suggestion below should map up to it.

```
    @Attribute(defaultValue = "true", dataType = Boolean.class)
    String getClusteringEnabled();
    public void setClusteringEnabled(String value);

```

```suggestion
            preBootCommands.add(new BootCommand("set", "configs.config.server-config.hazelcast-config-specific-configuration.clustering-enabled=false"));
```

Reply:
This still stands, I think this would be improved by adding the following to `HazelcastConfigSpecificConfiguration` class:

```
    @Attribute(defaultValue = "true", dataType = Boolean.class)
    String getClusteringEnabled();
    public void setClusteringEnabled(String value);
```

I can't do it as a suggestion since you haven't touched that class.

Once you've added the above, you can change this line to the following:
```suggestion
            preBootCommands.add(new BootCommand("set", "configs.config.server-config.hazelcast-config-specific-configuration.clustering-enabled=false"));
```